src/content-handlers/iiif/modules/uv-dialogues-module/ShareDialogue.ts

```diff
@@ -80,77 +74,77 @@ export class ShareDialogue<
       IIIFEvents.SHOW_EMBED_DIALOGUE,
       (triggerButton: HTMLElement) => {
         this.open(triggerButton);
-        this.openEmbedView();
+
+        if (triggerButton && $(triggerButton).is(".embed.btn")) {
+          // after setTimeout in Dialogue super class
+          setTimeout(() => {
```

Comment:
How important is this focusing behavior? Feels fragile to rely on a setTimeout if there's any other possible way...

Reply:
I find the focus to be accessibility and intuition critical, since we want to take users directly to the embed code. I agree that a timeout isn't ideal but I'm fighting [fire](https://github.com/UniversalViewer/universalviewer/blob/dev/src/content-handlers/iiif/modules/uv-shared-module/Dialogue.ts#L153) with slightly incremented fire.

Reply:
Makes sense. Do you think it's worth opening an issue about improving focus handling, so we don't lose track of the need to improve upon this (and maybe revisit default behavior, if any of it could be more accessible than it currently is)?